docs/env-vars.md

```diff
@@ -0,0 +1,10 @@
+# Environment variables management
+
+As a developer you will need to extend the current env vars for a service.
+
+The following rules must be followed:
+
+1. for each service that requires it, add it to the `services/docker-compose.yml` file (such as `MY_VAR=${MY_VAR}`)
+2. add a meaningful default value for development inside `.env-devel` so that developers can work, and that `osparc-simcore` is **self-consistent**.
```

Comment:
```suggestion
2. add a meaningful default value for development inside `.env-devel` so that developers can work, and that `osparc-simcore` is **self-contained**.
```

Reply:
you know what they say: "a typo a day keeps the refactor away"!